client/euphonia/src/App.css

```diff
@@ -1,4 +1,5 @@
 
```

Comment:
App.css is nice and organized. Good use of comments to sort out the sections

Reply:
I don't see all of App.css but I made these notes:
Line 262 -272, 367, 483 - these lines have been commented out, will you be keeping or deleting these lines?

Line 410, 438, 485, 621 - remove space on this line